src/cbc_sdk/platform/devices.py

```diff
@@ -254,6 +255,46 @@ def get_vulnerabilties(self):
         """
         return VulnerabilityQuery(Vulnerability, self._cb, self)
 
+    @property
+    def nsx_available(self):
+        """
+        Returns whether NSX actions are available on this device.
+
+        Returns:
+            bool: True if NSX actions are available, False if not.
+        """
+        return self._info['deployment_type'] == 'WORKLOAD' and self._info['nsx_enabled']
+
+    def nsx_remediation(self, tag, set_tag=True):
+        """
+        Start an NSX Remediation job on this device to change the tag.
+
+        Required Permissions:
+            appliances.nsx.remediation(EXECUTE)
+
+        Args:
+            tag (str): The NSX tag to apply to this device. Valid values are "CB-NSX-Quarantine",
+                       "CB-NSX-Isolate", and "CB-NSX-Custom".
+            set_tag (bool): True to toggle the specified tag on, False to toggle it off. Default True.
+
+        Returns:
+            NSXRemediationJob: The object representing all running jobs.  None if the operation is a no-op.
+        """
+        if not self.nsx_available:
+            raise ApiError("NSX actions are not available on this device")
```

Comment:
Quotes are inconsistent during all of the commits. I suspect that pre-commit is skipped. That also happens in `[nsx_remediation.py](https://github.com/carbonblack/carbon-black-cloud-sdk-python/pull/281/files#diff-466b2e4755f09b5752ef75bea324ba1ae06c6d52fc63684d004096bc4d7d42fd)` new additions.

Reply:
What do you mean by this? Also, your pre-commit stuff doesn't work on Windows.

Reply:
`'` and `"` are used at random there is no consistency. Also what does not work on pre-commit? I just installed it and used it without any problems on Windows. Eventually you can report an issue https://github.com/pre-commit/pre-commit here.

Reply:
OK, you're using some weird kind of pre-commit thing that I wasn't previously aware of.

Reply:
It says it's installed.

Reply:
I still see the deflake commits it seems to me that it's still not working. I don't understand the comment about the tool being weird but is there some error that you are getting? I would like to know so that I can raise an issue for it, thanks!

Reply:
Is there any way to run the quote-checker independent of the pre-commit mechanism, the way you can run flake8 independent of the pre-commit mechanism?

Reply:
There is no quote-checker in the `pre-commit` configuration. I just stated that as inconstancy, I think that we need to agree on a double/single quote strings and be consistent with it. 

If we stick to single quote we can add the `double-quote-string-fixer` step in the pre-commit configuration that would fix the double quoted stuff on each modified file in a commit.

Reply:
Resolving this topic since it is not in the scope of the PR.